docs/resources/Guild_Scheduled_Event.md

```diff
@@ -74,21 +120,32 @@ Create a guild scheduled event in the guild. Returns a [guild scheduled event](#
-| -------------------- | --------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------- |
-| channel_id?          | snowflake                                                                                                                   | the channel id of the scheduled event, if for a stage instance of voice channel |
-| entity_metadata?     | [entity metadata](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-entity-metadata) | the entity metadata of the scheduled event                                      |
-| name                 | string                                                                                                                      | the name of the scheduled event                                                 |
-| privacy_level        | [privacy level](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-privacy-level)     | the privacy level of the scheduled event                                        |
-| scheduled_start_time | ISO8601 timestamp                                                                                                           | the time to schedule the scheduled event                                        |
-| scheduled_end_time?  | ISO8601 timestamp                                                                                                           | the time when the scheduled event is scheduled to end                           |
-| description?         | string                                                                                                                      | the description of the scheduled event                                          |
-| entity_type          | [entity type](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-entity-types)        | the entity type of the scheduled event                                          |
+| Field                | Type                                                                                                                        | Description                                                                                |
+| -------------------- | --------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------ |
+| channel_id           | ?snowflake *                                                                                                                | the channel id of the scheduled event.                                                     |
```

Comment:
This is a breaking change. It should be noted that this is going from a "not required" to a "required null" option for External events. This isn't a huge issue but imo if possible it is cleaner to support `undefined` on the CREATE endpoint.